src/ui/editortabwidget.cpp

```diff
@@ -39,6 +40,18 @@ int EditorTabWidget::addEditorTab(bool setFocus, const QString &title)
     return this->rawAddEditorTab(setFocus, title, 0, 0);
 }
 
+bool EditorTabWidget::eventFilter(QObject* object, QEvent* ev)
+{
+    if(object == tabBar() && ev->type() == QEvent::FocusIn) {
```

Comment:
This fix could have been done in a different PR so that it would have been usable in the master branch too... Please don't mix different stuff in the same PR. There are a lot of reasons for not doing that.

I'll turn a blind eye for this time ;)